ski_conditions/apps/app_scraping/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class AppScrapingConfig(AppConfig):
```

Comment:
`app_scraping` is a bit redundant. Rename the app to `scraping`. This can be done now, or in a future change.

Reply:
I agree- I'll do it in the future.